webapp/src/shared/components/Avatar/EditableAvatar.tsx

```diff
@@ -54,13 +54,23 @@ export function EditableAvatar({
         onMouseOut={handleMouseUp}
         onMouseUp={handleMouseUp}
       >
-        <img
-          ref={ref}
-          src={url}
-          alt={url}
-          className="editable-avatar__image"
-          style={position}
-        />
+        {imgHash ? (
+          <img
+            ref={ref}
+            src={`${url}?${imgHash}`}
+            alt={url}
+            className="editable-avatar__image"
+            style={position}
+          />
+        ) : (
+          <img
+            ref={ref}
+            src={url}
+            alt={url}
+            className="editable-avatar__image"
+            style={position}
+          />
+        )}
```

Comment:
```suggestion
        <img
          onDragStart={(e) => e.preventDefault()}
          src={`${url}?${imgHash}`}
          alt={url}
          className="editable-avatar__image"
          style={position}
        />
```

suggestion: We could provide an inline `onDragStart` listener because react removes it automatically when the component unmounts and prevents memory leaks.

suggestion: We could change the type of the `imgHash` to string and provide an empty string as a default parameter.